packages/twitter-ui/my-app/src/App.scss

```diff
@@ -0,0 +1,95 @@
+$primary-color: #1DA1F2;
+$black: #000000;
+$gray: #8899A6;
+$white: #FFFFFF;
+
+
+
+
+.tweet {
+    background-color: $black;
+    width: 600px;  
+    font-family: 'Source Sans Pro';
+
+    &:not(:last-child) {
+        border-bottom: 2px solid $gray;
+    }
+
+
+    &__main-container {
+        padding: 10px 15px;
+        display: grid;
+        grid-template-columns: 50px 1fr;
+        column-gap: 10px;
+        grid-template-rows: repeat(1, 1fr);
+    }
+    &__full-name  {
+        color: $white;
+        margin: 0;
+        font-size: 1rem;
+        margin-right: 5px;
+    }
+
+    &__username {
+        color: $gray;
+        margin-left: 2px;
+        font-size: 12px;
+        margin-top: 2px;
+    }
+
+    &__user-container {
+        display: flex;
+
+    }
+     
+    &__message {
+        grid-column: 2/2;
+        color: white;
+        font-size: 12px;
+        margin-top: 5px;
+        margin-bottom: 12px;
+        grid-row-start: 2;
+        white-space: pre-line;
+    }
+
+    &__body {
+        color: $white
+    }
+
+    figure {
```

Comment:
It's usually better not to use html tags when following BEM, is there any reason we are using it?

Reply:
I just used it because the figure tag will not be permanent. I think it will probably be deleted in the future and be replaced with an image tag, but yes, I should have created a class as the image tag will share the most of the styling.